util/dashboard/gen_dashboard_entry.py

```diff
@@ -33,11 +35,103 @@ def genout(outfile, msg):
     'V3': 'Verification Complete'
 }
 
+# TODO: This is relative to the dashboard, which is currently located at
+# hw/_index.md.
+docs_server = "../.."
+
 
 def convert_stage(stagestr):
     return STAGE_STRINGS.get(stagestr, "UNKNOWN")
 
 
+# Link name with design spec
+def get_linked_design_spec(obj):
+    result = ""
+    if 'design_spec' in obj.keys():
+        url = docs_server + "/" + html.escape(obj['design_spec'])
+        result = "<span title='Design Specification'><a href=\"{}\">{}</a></span>".format(
+            url, html.escape(obj['name'].upper()))
+    else:
+        result = html.escape(obj['name'].upper())
```

Comment:
This changes IP names from lowercase in the old dashboard to ALL_CAPS. I don't really have preference here, but at the moment we quite inconsistently mix both in the documentation. Is there a plan to harmonize this at some point?

Reply:
As expressed in a previous mail, I really don't like the ALLCAPS; no need to yell the name of the IP block.

Reply:
Thanks @sriyerg .